.github/workflows/reviews.yml

```diff
@@ -0,0 +1,92 @@
+name: Reviews
```

Comment:
Should we have a comment summarising the purpose of the workflow?

> This workflow will request reviews from the respective owners and maintainers of each package listed in the PR's most recent `eval` comparison artifact.

Maybe my wording is slightly off, as I guess it isn't exclusive to _"package"_ owners, but any owned region of nixpkgs?